### ISSUE TITLE
Improper handling of legacy sites resulting in bright backgrounds 

### ISSUE THREAD

**Issue by Vintagemotors** [ThomazPom/Moz-Ext-UltimaDark#39]
Example site: https://www.sillycycle.com/abacus.html 

I will edit this to add more if I find others. 

Suggested behavior: disable image background and replace with background color from configuration chart or just a dark color.

Other than this your extension is nearly twice as fast as dark reader on first run for Speedometer 2.0 so well done!

**Comment by ThomazPom** [comment]
You are correct; I haven't yet made a decision regarding HTML4 legacy websites. I'm torn between transforming outdated tags from https://www.htmlhelp.com/reference/html40/deprecated.html into HTML5 tags or persisting in attempting to edit their color, which doesn't always prove effective as your examples shows.

I appreciate your comment about the load speed of UltimaDark, which I wasn't aware of. This is excellent news, as I had presumed UltimaDark might be slower due to the advanced techniques I mentioned in the ReadMe.md of this GitHub repository.

**Comment by Vintagemotors** [comment]
As an intermediary measure you could also consider creating a custom .css file to fallback on in cases where it is unable to change the color with your new methods. https://stackoverflow.com/questions/3099326/what-the-difference-between-the-support-for-css-by-html4-0-and-xhtml1-0

**Comment by Vintagemotors** [comment]
Fixed some time around 1.5.44
